tests/python/gaia-ui-tests/gaiatest/tests/functional/gallery/test_gallery_handle_invalid_file.py

```diff
@@ -14,7 +14,6 @@ class TestGalleryHandleInvalidPhoto(GaiaTestCase):
     # the test files are copied from https://github.com/mozilla-b2g/gaia/tree/master/apps/gallery/test/images by djf
     @parameterized("zero_length_file", 'image_formats/x05.png')
     @parameterized("text_file", 'image_formats/x06.jpg')
-    @parameterized("corrupt_jpg", 'image_formats/x07.jpg')
```

Comment:
There are intermittant timing failures in this test, but I think that I caused a permanent failure on x07.jpg.

That was a corrupt jpeg file that can be displayed by some tools and not by others. In 2.1, gecko does not display it. In 2.2, it displays it, and on master, recent changes seem to be causing an internal exception that was not thrown before.  I think this test was timing out because of the new gecko exception.

We should probably just let gecko handle images like this how it wants to, and this test is unfairly trying to force gecko to recognize bad images.